example/scalalib/publishing/9-cosmo-assembly/build.mill

```diff
@@ -0,0 +1,20 @@
+//// SNIPPET:BUILD
+package build
+import mill._, scalalib._
+
+object foo extends ScalaModule with cosmo.CosmoModule {
```

Comment:
Let's make this example a `JavaModule`, move it to `example/javalib/`, include it in `website/docs/modules/ROOT/pages/javalib/publishing.adoc`, and link to that section in `website/docs/modules/ROOT/pages/scalalib/publishing.adoc` and `website/docs/modules/ROOT/pages/kotlinlib/publishing.adoc`